src/IssueViz.Security.UnitTests/Hotspots/HotspotsList/HotspotsControlViewModelTests.cs

```diff
@@ -35,6 +36,13 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.Hotspots.
     [TestClass]
     public class HotspotsControlViewModelTests
     {
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            // HotspotsControlViewModel needs to be created on the UI thread
```

Comment:
Miscellaneous change to this and the tool window tests.
We must have been lucky with the test order execution before i.e. one of the earlier tests must have already called `SetCurrentThreadAsUIThread`.